build_tools/cmake/iree_copts.cmake

```diff
@@ -362,12 +352,23 @@ if(${IREE_SIZE_OPTIMIZED})
       "-opt:ref,icf"
   )
   # TODO(#898): make this only impact the runtime (IREE_RUNTIME_DEFAULT_...).
+  # These flags come from iree/base/config.h:
   set(IREE_DEFAULT_COPTS
       "${IREE_DEFAULT_COPTS}"
-      "${IREE_SIZE_OPTIMIZED_DEFAULT_COPTS}")
+      "${IREE_SIZE_OPTIMIZED_DEFAULT_COPTS}"
+      "-DIREE_STATUS_MODE=0"
+      "-DIREE_STATISTICS_ENABLE=0"
```

Comment:
Can we remove `-DIREE_STATISTICS_ENABLE=0`? We probably would still want to collect the statistics.